### PR TITLE
Processing transactions with the delayed inbox

### PR DESCRIPTION
**Pull request by timothymcmackin**
This is a rework of @vch9's submission in https://github.com/etherlinkcom/docs/pull/110. It builds on https://github.com/etherlinkcom/docs/pull/86.

Info about how the delayed inbox provides a backup method of getting transactions in that does not rely on the sequencer.

Preview: https://docs-etherlink-git-update-arch-diagram-add-de-59decb-trili-tech.vercel.app/network/architecture#delayed-inbox-transaction-processing

Added this diagram for the process, but not sure that we need it:
architecture-delayed-inbox

### CODE REVIEW

File: docs/network/architecture.md [hunk 1]
```diff
@@ -13,12 +13,27 @@ These components are instances of binaries in the [Octez software suite](https:/
...
-- It publishes transactions to EVM nodes, which can consider them final as long as they trust that the sequencer will publish them to layer 1.
-- It publishes transactions to Tezos layer 1, which finalizes them.
+Each blueprint contains:
+
+- A list of transactions
+- A list of transactions that are currently in the delayed inbox, as described in [Transaction lifecycle](#transaction-lifecycle)
+- The hash of the previous blueprint
+- The timestamp of the blueprint.
+
+You can imagine blueprints as the blocks in Etherlink's layer 2 chain.
```
Comment (vch9) [inline docs/network/architecture.md:25]: Not sure about this, but I'm not sure yet what to propose instead

File: docs/network/architecture.md [hunk 2]
```diff
@@ -13,12 +13,27 @@ These components are instances of binaries in the [Octez software suite](https:/
...
+The sequencer publishes each blueprint in two ways:
+
+- It publishes them to EVM nodes, which can consider the transactions final as long as they trust that the sequencer will publish them to layer 1.
+- It publishes them to Tezos layer 1, which finalizes the transactions.
+
+The sequencer is the primary way that Etherlink transactions are processed.
+However, to protect the system from censorship and any other problems with the sequencer, Etherlink provides a backup way of handling transactions; see [Transaction lifecycle](#transaction-lifecycle).
 
 The sequencer is an instance of the `octez-evm-node` binary running in sequencer mode.
+Only members of the Sequencer Committee can run instances of the sequencer in sequencer mode.
```
Comment (vch9) [inline docs/network/architecture.md:36]: there's no committee yet, it might be confusing

File: docs/network/architecture.md [hunk 3]
```diff
@@ -51,12 +66,16 @@ For more information about Tezos layer 1 and its nodes, see [Architecture](https
 
 ## Transaction lifecycle
 
-The overall lifecycle of a typical operation is as follows:
+Etherlink has a standard method of processing transactions and a backup method that protects it from censorship and
```
Comment (vch9) [inline docs/network/architecture.md:69]: missing end of sentence

File: docs/network/architecture.md (hunk 1 above)
Comment (timothymcmackin) [reply docs/network/architecture.md:25]: This could be confusing -- removed it.

File: docs/network/architecture.md (hunk 3 above)
Comment (timothymcmackin) [reply docs/network/architecture.md]: Completed it as "protects it from censorship and network problems."

File: docs/network/architecture.md (hunk 2 above)
Comment (timothymcmackin) [reply docs/network/architecture.md:34]: We talk about the Sequencer Committee in the governance info, so I'm comfortable using that term here. We could remove this sentence if it is confusing; it's just some extra info that I thought would be helpful.

**Pull request merged**